kudulite/startup.sh

```diff
@@ -50,4 +50,4 @@ export APPDATA=/opt/Kudu/local
 cd /opt/Kudu
 
 echo $(date) running .net core
-ASPNETCORE_URLS=http://0.0.0.0:"$PORT" runuser -p -u "$USER_NAME" -- benv dotnet=2.2 dotnet Kudu.Services.Web.dll
+ASPNETCORE_URLS=http://0.0.0.0:"$PORT" runuser -p -u "$USER_NAME" -- benv dotnet=3.1 dotnet Kudu.Services.Web.dll
```

Comment:
Kudu for carrying out git deploys spawns a Kudu.Console process. I'm not sure if we support Git Deploys with Consumption functions but there is a change in `3.1 Migration` where for linux since Kudu is a self contained app it  would spawn `/opt/Kudu/Kudu.Console/kudu` previously this was `benv dotnet=2.2 /opt/Kudu/Kudu.Console/kudu.dll` since here it is not a self contained app that might fail

Reply:
Discussed offline, Linux Consumption does not support [Git Deploy](https://docs.microsoft.com/en-us/azure/azure-functions/functions-deployment-technologies#deployment-technology-availability) and Console for now.